csharp-basic-topics/GuidClassInCSharp/GuidClassInCSharp/Methods/GuidToStringMethod.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace GuidClassInCSharp
+{
+    public partial class GuidClassInCSharpMethods
+    {
+        public static string GuidToString(Guid guid)
```

Comment:
Guid.ToString takes format arguments, it would be good to add an additional format parameter to show the options available with a call to ToString (i.e. ToString("X") or ToString("N"))